### ISSUE TITLE
DEV: Update create template flow

### ISSUE THREAD

**Issue by yaelberger-commits** [cds-snc/notification-planning#742]
_Update create template flow_

## Description

As a new or newer Notify user, I need to understand what a template is in the Notify product flow, and when and how to create one, so that I can successfully create and send a notification to one or many people without getting confused or abandoning my task. 

## Acceptance Criteria** (Definition of done) 

_To be refined through discussion with the team_

Given a new or returning Notify user needs to create a template to send a message, when they are signed in and going through the send message flow, they understand the concept of a template, how and when to create one without confusion or frustration and that the content they see matches the Figma files in both EN and FR.

- [x] Change two headings when you choose the type of message (email or text) and on the "create a template" page
- [x] Make sure all content changes in Figma are made in Staging and Prod for both EN and FR
- [x] make use of "help card" component in issue #740 (dependency)

## QA Steps to be completed by a second peer reviewer, doesn't have to be a developer

- [x] Content in Staging and Prod matches content in Figma for both EN and FR
- [ ] A11y manual tests
- [ ] Check that any links are going to the right places and are working
- [x] Check that headings are labelled correctly for a11y
- [x] test content in all browsers (Chrome, Safari, Firefox)
- [ ] check different resolution for responsiveness in Chrome (potentially using [Responsively app](https://responsively.app/))
- [x] check in mobile

**Comment by amazingphilippe** [comment]
Figma: https://www.figma.com/file/RLcM0JwlQsd2uVK3GJr5qY/Drafting-message-templates-%7C-Creer-des-gabarits-de-message?node-id=2102%3A11366

**Comment by yaelberger-commits** [comment]
Phil will make sure it's ready for review

**Comment by amazingphilippe** [comment]
Needs a reviewer

**Comment by jzbahrai** [comment]
@andrewleith to QA

**Comment by andrewleith** [comment]
Here are some notes.  Most tests passed but there are a few things left to fix:

- ❌  A11y manual tests
   - id referenced in `aria-labelledby` attribute on `fieldset#process_type` doesn't exist
   - Note: `legend` elements already describe fieldsets, so this may not be necessary at all
- ❌ Check that any links are going to the right places and are working
   - Content exists and links are good but they don't open in a new tab
- ❌ check different resolution for responsiveness in Chrome 
   -  Form content doesn't take up full width of available space 
   - <img width="1064" alt="image" src="https://user-images.githubusercontent.com/823749/183462647-102e2b5d-7a66-468d-8abf-18465541ccd7.png">

**Comment by amazingphilippe** [comment]
For Andrew's QA points
1. process_type is the priority setting visible to us admins. Not our users. I suggest not fixing for now
2. Fixed the external links to open in a new tab! Good catch!
3. We want to make sure the width of each line is shorter than 80 characters wide for legibility. This explains the white space. I added a bit of spacing to the cards to let the layouts breathe individually.
